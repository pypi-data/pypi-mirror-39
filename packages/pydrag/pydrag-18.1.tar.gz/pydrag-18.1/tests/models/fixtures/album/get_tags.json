{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?album=A+Night+at+the+Opera&api_key=LAST_FM_API_KEY&artist=Queen&autocorrect=1&format=json&mbid=6defd963-fe91-4550-b18e-82c685603c2b&method=album.getTags&user=Zaratoustre"
            },
            "response": {
                "body": {
                    "string": "{\"tags\":{\"tag\":[{\"name\":\"bar\",\"url\":\"https://www.last.fm/tag/bar\"},{\"name\":\"foo\",\"url\":\"https://www.last.fm/tag/foo\"}],\"@attr\":{\"artist\":\"Queen\",\"album\":\"A Night at the Opera\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "178"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:39 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
