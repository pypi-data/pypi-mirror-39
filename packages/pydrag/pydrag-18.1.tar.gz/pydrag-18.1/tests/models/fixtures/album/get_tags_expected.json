{
    "album": "A Night at the Opera",
    "artist": "Queen",
    "data": [
        {
            "name": "bar",
            "url": "https://www.last.fm/tag/bar"
        },
        {
            "name": "foo",
            "url": "https://www.last.fm/tag/foo"
        }
    ]
}
